src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/HostWorkspaceServicesExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.Host
+{
+    internal static class HostWorkspaceServicesExtensions
+    {
+        public static TLanguageService? GetLanguageService<TLanguageService>(this HostWorkspaceServices hostWorkspaceServices, string languageName) where TLanguageService : class, ILanguageService
```

Comment:
This file has all new code, which is added to ensure that GetLanguageService when invoked from CodeStyle layer finds the appropriate language services implemented in CodeStyle layer.